### DISCUSSION TITLE
redundant translations

### DISCUSSION THREAD

**Discussion by majdiriahi** [vinissimus/next-translate#622] [General] [answered]
How can i associate both of languages **fr** and **fr-be** to the same translation file **locales/fr/common.json** to avoid redundancy ?

**Comment by aralroca** [comment, marked as answer]
You need to overwrite the `loadLocaleFrom` in the `i18n.js` configuration.

```js
module.exports = {
  ...config,
  loadLocaleFrom: (lang, ns) =>
    import(`./locales/${lang.split('-')[0]}/${ns}.json`).then((m) => m.default),
}
```
